src/installer.ts

```diff
@@ -32,12 +32,6 @@ export async function getBuildx(version: string): Promise<string> {
   return path.join(cliPluginsDir, pluginName);
 }
 
-function getCliPluginsDir(): string {
-  const cliPluginsPath = path.join(os.homedir(), '.docker', 'cli-plugins');
-  fs.mkdirSync(cliPluginsPath, {recursive: true});
-  return cliPluginsPath;
-}
-
```

Comment:
My IDE told me it was not used, I double-checked and it's indeed not used anymore.

I've deleted it because of the `os.homedir(), '.docker'` which bothered me as it was not in the main function's scope.